### PR TITLE
HTML writer: render inline formulas correctly

### LINKED ISSUE #5655: --webtex uses display style regardless of delimiters
The web service used by Pandoc to convert TeX formulas to images uses display style by default. I don't know if the service has a parameter for specifying the style. A workaround is to prefix inline formulas with `\textstyle` to force it to render using inline style.

**Version:**
```sh
$ pandoc --version
pandoc 2.7.4
```

**Reproduce:**
```sh
$ pandoc --webtex
$\frac ab$
^D
```

**Expected:** URL to image of inline style formula.

**Actual:** URL to image of display style formula.

### PR DESCRIPTION
**Pull request by pesca**
Closes #5655.

### CODE REVIEW

**Comment by jgm** [comment]
Good idea!

**Comment by pesca** [comment]
@jgm I feel the assumptions about the web service needs to be documented in the manpage. Otherwise a service which renders using inline style by default will not correctly render display formulas. Another alternative is to make no assumptions and prefix display formulas with `\displaystyle`. I'd be glad to make whichever change you feel is better.

File: src/Text/Pandoc/Writers/HTML.hs [hunk 1]
```diff
@@ -1062,8 +1062,11 @@ inlineToHtml opts inline = do
       case writerHTMLMathMethod opts of
            WebTeX url -> do
               let imtag = if html5 then H5.img else H.img
+              let s = case t of
+                           InlineMath  -> "\\textstyle "
+                           DisplayMath -> ""
```
Comment (pesca) [inline src/Text/Pandoc/Writers/HTML.hs]: Prefix display formulas in case an alternative service uses inline style by default?

Comment (jgm) [reply src/Text/Pandoc/Writers/HTML.hs]: This probably makes sense.

**Pull request merged**